packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/textbox.tests.js

```diff
@@ -220,6 +220,20 @@ QUnit.module('options changing', {
         }
     });
 
+    QUnit.test('The validation error should be null after removing the mask', function(assert) {
```

Comment:
add please bug ID to test's name

Reply:
[3f3edd83908ad2052b6a025e4c3f30a35f5c9354]